src/Form/input-classifiers.test.js

```diff
@@ -38,48 +38,6 @@ afterEach(() => {
 })
 
 describe('Input Classifiers', () => {
-    it('should match the selector for cardNumber', () => {
```

Comment:
This was the initial basic test. It's not really needed, considering the more extensive testing below.